PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/BrowserSwitchHelper.kt

```diff
@@ -40,4 +44,16 @@ internal class BrowserSwitchHelper(private val urlScheme: String) {
             .returnUrlScheme(urlScheme)
             .metadata(metadata)
     }
+
+    fun configurePayPalVaultApproveSwitchOptions(
+        setupTokenId: String?,
+        approveOrderHref: String
+    ): BrowserSwitchOptions {
+        val metadata = JSONObject()
+            .put(METADATA_KEY_SETUP_TOKEN_ID, setupTokenId)
+        return BrowserSwitchOptions()
```

Comment:
What is this metadata for Steven? In iOS I am just taking the string from url and launching a webView.